lib/internal/inspector/load_network_resource_worker.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+const {
+  JSONParse,
+  JSONStringify,
+} = primordials;
+const {
+  emitProtocolResponseInParent,
+  addIoData,
+} = internalBinding('inspector');
+const { AbortController } = require('internal/abort_controller');
+const { setTimeout, clearTimeout } = require('timers');
+const { fetch } = require('internal/deps/undici/undici');
+
+module.exports = function setupInspectorWorker(_, port) {
+  function emitResponse(callId, sessionId, success, streamId = null) {
+    const result = {
+      id: callId,
+      result: {
+        resource: {
+          success,
+        },
+      },
+    };
+    if (success && streamId !== null) {
+      result.result.resource.stream = streamId.toString();
+    }
+    emitProtocolResponseInParent(callId, JSONStringify(result), sessionId);
+  }
+
+  port.on('message', (msg) => {
+    const { sessionId, callId, message } = msg;
+    const url = JSONParse(message)?.params?.url;
+    if (!url) return;
+
+    const controller = new AbortController();
+    const timeoutId = setTimeout(() => controller.abort(), 2000);
+
+    fetch(url, { signal: controller.signal })
```

Comment:
Can we limit the origin of the url to be loopback interface like `localhost` only? This could reduce the security interface that this feature exposes.